src/calendar.h

```diff
@@ -74,17 +77,24 @@ class calendar
         bool      operator ==(int rhs) const;
         bool      operator ==(const calendar &rhs) const;
 
-        void increment();   // Add one turn / 6 seconds
+        /** Increases turn_number by 1. (6 seconds) */
+        void increment();
 
         // Sunlight and day/night calculations
-        int minutes_past_midnight() const; // Useful for sunrise/set calculations
-        moon_phase moon() const;  // Find phase of moon
-        calendar sunrise() const; // Current time of sunrise
-        calendar sunset() const;  // Current time of sunset
-        bool is_night() const;    // After sunset + TWILIGHT_MINUTES, before sunrise
-        int sunlight() const;     // Current amount of sun/moonlight; uses preceding funcs
-
-        // Basic accessors
+        /** Returns the number of minutes past midnight. Used for sunrise/set calculations. */
+        int minutes_past_midnight() const;
```

Comment:
Needs a doomsday clock note; nuclear weapons were used on the portals and likely accelerated the Cataclysm.  ;-)